conf/messages

```diff
@@ -1,5 +1,12 @@
 # https://www.playframework.com/documentation/latest/ScalaI18N
 edit-set.heading=Edit set: {0}
+edit-set.ordering.button=Sort edit set
```

Comment:
@rwalpole, as [requested](https://tna-digital.slack.com/archives/CQB3ETSS3/p1672850299572729?thread_ts=1672843427.388509&cid=CQB3ETSS3), I changed the button from "Go".